### PR TITLE
Instructor: help page: remove link-style formatting from sub-titles #8190

### LINKED ISSUE #8190: Instructor: help page: remove link-style formatting from sub-titles
Description of feature/enhancement
Within the help document, various subtitles show up with a blue line underlining/highlighting them as if they are a hyperlink to click on and retrieve more information from. This could present confusion for users who may think they are missing additional information or be looking for a link to follow. 

Attached below are two screenshots that show the hi-lighting area I am referring to.

help1

help2

The enhancement would remove this hi-lighting/underlining when hovering over a subtitle. 

Justification
This will make the application more user friendly and easier for the user to understand.

### PR DESCRIPTION
**Pull request by craaaa**
Fixes #8190

**Outline of Solution**
- Moved link bookmarks from `<a>` tag into `<li>` or `<h>` tags where relevant to match [`studentHelp.jsp`](http://teammatesv4.appspot.com/studentHelp.jsp)

Demo can be seen [here](https://teammates-cara.appspot.com/instructorHelp.jsp)

### CODE REVIEW

**Comment by craaaa** [comment]
Noticed stylistic inconsistencies in `instructorHelp.jsp` while doing this:
- Paragraph spacing using `<br>` tags not consistent -- should perhaps use `<p>`?
- Inconsistent use of heading styles and numbering (compare FAQ and Sections)
- Missing `example` divs around some examples

Decided not to fix it in this PR since it wasn't part of the original scope -- should I open a new issue instead?

**Comment by damithc** [comment]
> Decided not to fix it in this PR since it wasn't part of the original scope -- should I open a new issue instead?

Yup, can open a new issue @craaaa

File: src/main/webapp/instructorHelp.jsp [hunk 1]
```diff
@@ -27,13 +27,13 @@
...
-              <a href="#faq6">How do I change the Google ID associated with a student?</a>
+              <a href="#faq5">How do I change the Google ID associated with a student?</a>
             </li>
             <li>
-              <a href="#faq7">What to do if a student says his/her courses have disappeared from the system?</a>
+              <a href="#faq6">What to do if a student says his/her courses have disappeared from the system?</a>
             </li>
             <li>
-              <a href="#faq7a">How do I interpret/use contribution values in the results of ‘Team contribution’ type questions?</a>
+              <a href="#faq7">How do I interpret/use contribution values in the results of ‘Team contribution’ type questions?</a>
```
Comment (xpdavid) [inline src/main/webapp/instructorHelp.jsp]: Be careful when you change the name. I notice that you have change the `<li>`'s id accordingly. However, other html tag also refer to these id. Check these :)
image
image

**Comment by craaaa** [comment]
@xpdavid ah, forgot about that! made the necessary changes :)
I searched the codebase for `faq7` and found no references to the original `faq7`, then changed all references to `faq7a` to `faq7`.

**Comment by xpdavid** [comment]
Fetched and run locally. LGTM. Good work @craaaa. You may update your demo for @damithc to review.

**Pull request merged**